src/http-adapter.js

```diff
@@ -76,32 +80,39 @@ class HttpAdapter {
    * @returns {Promise} Promise for destroy.
    */
   destroy (link) {
-    return this._ajax('DELETE', link)
+    return this._http(METHOD.DELETE, link)
   }
 
-  _ajax (type, url, data) {
-    let headers = {
-      'Accept': 'application/vnd.api+json',
-      'Content-Type': 'application/vnd.api+json'
+  async _http (method = '', url = '', data = {}) {
```

Comment:
What does the fetch API/polyfill do if you give it an empty string as the method and/or URL?

Reply:
actually havent evaluated that since we havent handled a default method in the past. I wasnt sure if it would be an issue or not since it is a private method. Should we add one?

Reply:
I think we either need a reasonable default or a reasonable error. My concern is that by defaulting to an empty string, the code might fail in a way that obscures the root cause.

Reply:
good point. I'm thinking of defaulting the method to 'GET' and if the url is empty throwing an error